ci/_main.sh

```diff
@@ -105,6 +105,7 @@ function runInBuilder() {
         --env TZ="Europe/Moscow" \
         --env LOCAL_USER_ID="$USER_ID" \
         --env BINTRAY_GPG_PASSPHRASE="$BINTRAY_GPG_PASSPHRASE" \
+        --env GRADLE_USER_HOME=/gradle \
```

Comment:
This env is optional. Should we consider it?

Reply:
You mean we should fail on it's abscence in entrypoint?

Reply:
This is an option. 
The alternative is to repeat default behavior: use `$USER_HOME/.gradle` if not set.

Reply:
There is a problem with mounting into folder of not yet created user; 
i didn't go deeper, please check `entrypoint.sh` changes